pyproject.toml

```diff
@@ -6,3 +6,22 @@ requires = [
     "Cython>=0.29",
     "oldest-supported-numpy",
 ]
+
+[tool.cibuildwheel]
+skip = ["cp36-*", "cp37-*", "pp*"]
+
+[tool.cibuildwheel.windows]
+archs = ["auto64"]
+
+before-build = "pip install delvewheel"
+repair-wheel-command = "delvewheel repair --add-path C:/vcpkg/installed/x64-windows/bin -w {dest_dir} {wheel}"
```

Comment:
I didn't directly find a way to use an env variable here, so this `-add-path` is currently hardcoded (in practice, I don't think it will be common that people use this config to build wheels locally using cibuildwheel, so that's probably not such a problem)

Reply:
This seems fine.